src/manage_nsfs/manage_nsfs_validations.js

```diff
@@ -91,6 +91,8 @@ function validate_no_extra_options(type, action, input_options, is_options_from_
         valid_options = VALID_OPTIONS.account_options[action];
     } else if (type === TYPES.GLACIER) {
         valid_options = VALID_OPTIONS.glacier_options[action];
+    } else if (type === TYPES.ANONYMOUS_ACCOUNT) {
+        valid_options = VALID_OPTIONS.anonymouse_account_options[action];
```

Comment:
If it helps you in case you remove the `TYPE.ANONYMOUS_ACCOUNT` we have the `from_file_condition`, you can do something similar.